addons/addon-base-raas-ui/packages/base-raas-ui/src/models/aws-accounts/AwsAccount.js

```diff
@@ -27,6 +27,9 @@ const AwsAccount = types
     description: '',
     accountId: '',
     externalId: '',
+    needsPermissionUpdate: true,
```

Comment:
Is there a `types.boolean` ?

Reply:
there is, although setting it to `true` has the same effect with a default value of `true`.